wurst/data/HashMap.wurst

```diff
@@ -104,9 +119,11 @@ public class IterableMap<K,V> extends HashMap<K, V>
 
 
 	/** Returns the length of this IterableMap **/
-	function size() returns int
+	override function size() returns int
 		return keys.size()
 
+	function mapSize() returns int
```

Comment:
why is this here? size() and mapSize() should always be the same, right?

Reply:
yes. and i use it in the hashmaptest.wurst to check the size . it doesn't make any sense. Do I need to get rid of it?

Reply:
Yes